mail-filter/imapfilter/imapfilter-2.7.6.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..4} luajit )
+
+inherit lua-single toolchain-funcs
+
+DESCRIPTION="An IMAP mail filtering utility"
+HOMEPAGE="https://github.com/lefcha/imapfilter"
+SRC_URI="https://github.com/lefcha/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+REQUIRED_USE="${LUA_REQUIRED_USE}"
+
+RDEPEND="
+	dev-libs/openssl:0=
+	dev-libs/libpcre2
+	${LUA_DEPS}"
+DEPEND="${RDEPEND}"
+
+DOCS="AUTHORS NEWS README samples/*"
+
+src_prepare() {
+	default
+	sed -i -e "/^PREFIX/s:/usr/local:${EPREFIX}/usr:" \
```

Comment:
This should likely be a patch but I won't insist you do it this round.

Reply:
done anyway :)

Reply:
However I'm not sure that's actually correct since EPREFIX is obviously pre-resolved in the patch, not at build time. Or maybe I misunderstood what you wanted..?

Reply:
Did you maybe mean to put `$(EPREFIX)/usr` (note round brackets) into the patch so that the ebuild/emake can pass the content of ${EPREFIX} to make, which then does the substitution?

Reply:
I've reverted the patch and reinstated the sed expression for now.

Reply:
It should take it as a variable instead.